scalding-spark/src/main/scala/com/twitter/scalding/spark_backend/SparkWriter.scala

```diff
@@ -127,6 +127,9 @@ class SparkWriter(val sparkMode: SparkMode) extends Writer {
   def getIterable[T](conf: Config, initial: TypedPipe[T])(implicit cec: ExecutionContext): Future[Iterable[T]] =
     state.get().getIterable(conf, initial)
 
+  def getCleanupFor(uuid: UUID)(implicit cec: ExecutionContext): Future[Unit] =
+    Future.successful(())
```

Comment:
I think spark has the same issue: giant data may stick around for a long time, but we can ignore for now.

Reply:
ok, got it.